kivy/tests/test_window_base.py

```diff
@@ -18,3 +18,26 @@ def test_to_normalized_pos(self):
                 assert result_sy == expected_sy
         finally:
             win.system_size = old_system_size
+
+
+class WindowOpacityTest(GraphicUnitTest):
+
+    def test_window_opacity_property(self):
+        from kivy.core.window import Window
```

Comment:
You can use `self.Window` to access window instance, so no need for import statement.